Formula/diffoscope.rb

```diff
@@ -30,7 +31,6 @@ class Diffoscope < Formula
     sha256 "9919344cec203f5db6596a29b5bc26b07ba9662925a05e24980b84709232ef60"
   end
 
-  # required by cmdline
```

Comment:
Is this dependency still relevant for `cmdline`?

Reply:
`argcomplete` and `progressbar` are both required by `cmdline`, but everything is found if you grab the resources for `diffoscope[cmdline]` which is why I changed the comment at the top. I'm not sure which comment is more clear though.

Reply:
Maybe something like `# Use resources from diffoscope[cmdline]`?